concepts/05 Widgets/DataGrid/20 Editing/20 API/20 Update.md

```diff
@@ -103,6 +194,78 @@ The **DataGrid** widget allows you to process an updated cell value in the **col
         <dxi-column dataField="ID" [visible]="false"></dxi-column>
         <dxi-column dataField="Full_Name" [setCellValue]="setCellValue"></dxi-column>
     </dx-data-grid>
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDataGrid ... >
+            <DxEditing
+                :allow-updating="true"
+                :allow-adding="true"
+            />
+            <DxColumn data-field="ID" :visible="false" />
+            <DxColumn data-field="Full_Name" :set-cell-value="setCellValue" />
+        </DxDataGrid>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxDataGrid, {
+        DxEditing,
+        DxColumn
+    } from 'devextreme-vue/data-grid';
+
+    export default {
+        components: {
+            DxDataGrid,
+            DxEditing,
+            DxColumn
+        },
+        methods: {
+            setCellValue(rowData, value) {
```

Comment:
Currently, the https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#setCellValue article has better example for setCellValue with more real scenario and with using the third argument currentRowData. Can we copy it to here?

Reply:
The example is removed, and the user is advised to refer to the setCellValue description.